logger/src/logger.cpp

```diff
@@ -56,4 +57,36 @@ void logMessage(const std::string& message,
     logMessage(Loglevel::INFO, message, location);
 }
 
+void fetchNLogs(size_t noOfLogs)
+{
+    try
+    {
+        std::fstream logfile("/tmp/logdata.txt",
```

Comment:
can you we keep this log file path in header as a const variable.
and usually log files will end with .log extension.

Reply:
Constant is ok, but I don't agree with moving it to the header file.
Since the use is local to the file, declare in the scope of this file itself.

Reply:
done